client/a8c-for-agencies/sections/sites/features/jetpack/activity/style.scss

```diff
@@ -1,3 +1,7 @@
 .activity-log-v2 {
 	text-align: left;
 }
+
+main.activity-log-v2:not(.wordpressdotcom) .activity-log-v2__header {
```

Comment:
In the PR I'm working on, this (`.wordpressdotcom`) is going to be removed as it is added as a classname purely because we're not in the Jetpack Cloud environment (hence it's assumed we're in WP.com). No other class is added, but I could add one if it would be helpful. Unless it's possible to avoid using the `.wordpressdotcom` here and still achieve the same effect?

Reply:
https://github.com/Automattic/wp-calypso/pull/89388/commits/6a655b8b000cc872f5bc8ef5047de8a55333ea83